handbook/engineering/code-graph/index.md

```diff
@@ -0,0 +1,14 @@
+# Code Graph engineering org
+
+The Code Graph org's mission is to build the **code graph**.
+Each team within our org implements a different _verb_ on the code graph:
+
+- The [Search team](./search/index.md) enables _searching_ the code graph
```

Comment:
could we add the following nesting below this line and change "Search team" to "Search teams" – 

```
    - [Search Core team](./search/core/index.md)
    - [Search Product team](./search/product/index.md)
```

Reply:
Good idea

Reply:
Done